### ISSUE TITLE
GitHub authentication issues

### ISSUE THREAD

**Issue by reillydavis** [reenhanced/gitreflow#193]
I'm having issues with GitHub authentication. Whenever I run a reflow command, it fails and gives the message below. If I delete the token from my GitHub account, it works one time, but then will come with the same error the next time. Using two step authentication with GitHub so not sure if that has anything to do with it. Any ideas? Thanks!

Using Ubuntu 16.04 with reflow version 0.8.1

`reilly@reilly:~/Documents/GitHub/campuskudos-api$ git reflow status`
`Please enter your GitHub username: reillydavis`
`Please enter your GitHub password (we do NOT store this):`
`Please enter your two-factor authentication code: *******`
`[error] You have previously setup git-reflow on this machine, but we can no longer find the stored token.`
`[notice] Please visit https://github.com/settings/tokens and delete the token for: git-reflow (reilly)
error: We were unable to authenticate with Github.`

**Comment by codenamev** [comment]
I know you're on Ubuntu, but we recently changed the way we install and reference git-reflow specific git configurations (see if #187 solves your issue).

It is most likely an issue of blank keys in one of your git-config files.  Check all config files in `~/.gitconfig`, `~/.gitconfig.reflow`, `your/project/path/.git/config` clear out any github/reflow sections and re-run `git reflow setup`.

Sorry for the issues here!  We are working on smoothing out upgrades transitioning to the new config setup.

**Comment by codenamev** [comment]
@reillydavis are you still seeing issues here?

**Comment by codenamev** [comment]
If you still have issues re-open.  I may reference this in future tasks that may auto-fix the config disconnect you're seeing.

**Comment by abhishek77in** [comment]
I am having similar issue. No matter what I do I am not able to successfully setup the reflow again. The issue has existed for so long for me, I have given up on this twice.

I have tried to remove git-reflow from all my git config files, still I am unable to setup successfully.  I keep getting message that your GitHub account was already setup. 

I have used git-reflow in past, this may be due to older installation (4 year plus). Let me know if anyone can help.

Attaching terminal screenshot for reference. 

![1__abhishek_abhisheks-iMac____labsmart-lis__zsh_](https://user-images.githubusercontent.com/749134/64765145-3fc61180-d561-11e9-8f84-a70b0ed4e5f8.jpg)
